internal/handlers/sqlite/msg_create.go

```diff
@@ -108,3 +119,29 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, lazyerrors.Error(err)
 	}
 }
+
+// validateCollectionNameRe validates collection names.
+// Empty collection name, names with `$` and `\x00` are not allowed.
+// Collection names that start with `.` are also not allowed.
+var validateCollectionNameRe = regexp.MustCompile("^[^.$\x00][^$\x00]{0,234}$")
```

Comment:
That's an interesting place to define a new variable. Do we have a convention where new variables should be defined? E.g. you didn't define it inside a function, but on a package level, on the other hand, you defined it in the middle of the file. If it's used inside a particular function only, maybe we don't need it on a package level? Or if we need it on a package level (why?) it should be on the top of the file where it's used?